script/install-as-user

```diff
@@ -127,6 +127,9 @@ sed -r \
     -e "s,# constraint_disabling: false,  constraint_disabling: false," \
     config/database.yml-example > config/database.yml
 
+# create the template_utf8 template we'll use for our databases
+createdb -T template0 -E UTF-8 template_utf8
```

Comment:
I had to [explicitly set](http://alaveteli.org/docs/installing/manual_install/#configure-database) `datistemplate`. Not sure how much it matters